packages/n/nana/xmake.lua

```diff
@@ -18,6 +21,7 @@ package("nana")
         add_defines("_SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING")
     elseif is_plat("linux") then
         add_syslinks("pthread", "X11", "Xft", "fontconfig")
+        add_links("nana")
```

Comment:
we need not add it, xmake will find links in lib directory

Reply:
Yes it should do this. But for some reason it didn't on my system

Reply:
If the .a library exists in the installdir/lib directory, you should be able to find this links

We should not add it. If you can't find links on your machine, you should open a bug issue.

Reply:
cmake creates nothing inside the install directory. I think nana doesn't support an install prefix.

Reply:
I tested it, and even if this link is not set, xmake can automatically find the links.

As long as you copy libnana.a to the lib directory, xmake will find it automatically, you don’t need to add `add_links("nana")`

It was not found on your machine before, maybe you haven't added `os.cp("*.a", "lib")` at the beginning, so there is no library file in the lib directory, so xmake didn't add nana links

Reply:
You are right. I removed it